.github/workflows/tests.yaml

```diff
@@ -609,7 +609,7 @@ jobs:
                      --app out/linux-x64-all-clusters-ipv6only-no-ble-no-wifi-tsan-clang-test/chip-all-clusters-app \
                      --app-args "--discriminator 3840 --interface-id -1" \
                      --tool-path out/linux-x64-java-matter-controller \
-                     --tool-cluster "pairing" \
+                     --tool-cluster "im" \
```

Comment:
I agree those IM commands (read/write/subscribe/invoke) should not under 'pairing' cluster, but it looks they are still implemented as the commands within the 'pairing' cluster

Reply:
removed. thanks